lametro/templates/lametro/index.html

```diff
@@ -7,8 +7,8 @@
 {% endblock %}
 
 {% block full_content %}
-    <div class="container-fluid" id="section-photo">
-        <div class="row-fluid">
+    <div id="section-photo">
```

Comment:
How about retaining `.container-fluid` to preserve padding on smaller screens?

```suggestion
    <div id="section-photo" class="container-fluid">
```

Without:

![Screen Shot 2021-04-20 at 1 46 21 PM](https://user-images.githubusercontent.com/12176173/115448048-d4353300-a1de-11eb-870a-6183f8f7b5b3.png)

With:

![Screen Shot 2021-04-20 at 1 45 13 PM](https://user-images.githubusercontent.com/12176173/115448057-d7c8ba00-a1de-11eb-86f4-75808157b45d.png)